websites/V/Vodafone FM/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "180867904135233536",
+		"name": "AnOmniMoose#3670"
+	},
+	"service": "Vodafone FM",
+	"description": {
+		"en": "Vodafone FM Portuguese radio"
```

Comment:
Off the bat, this doesn't seem like an **official** description of the service

Reply:
The website uses "Mexe na Música"; "Sabe tudo sobre as iniciativas da Vodafone na música!" is the bio of their Instagram account.

Reply:
Good point! Took the Wikipedia description, simplified it and translated it in https://github.com/PreMiD/Presences/pull/7113/commits/f23efd53add85f3d3bce061b98fdc755d0abc9dd